packages/docusaurus-theme-classic/src/theme/DocSidebar/index.tsx

```diff
@@ -109,7 +106,7 @@ function DocSidebarItemCategory({
           'menu__link--active': collapsible && isActive,
           [styles.menuLinkText]: !collapsible,
         })}
-        onClick={collapsible ? handleItemClick : undefined}
+        onClick={collapsible ? () => handleItemClick(item) : undefined}
```

Comment:
this may create some performance cost because it re-creates arrow functions on every render. Especially after the removal of `useCallback` in `handleItemClick`. You may want to look into how to get useCallback hook back, or just into options how to optimize this

Reply:
I don't think it's an issue, as it's not a custom/expensive React comp using memorization.

This useCallback actually may be useless in the first place, ref https://royi-codes.now.sh/thousand-usecallbacks/